PyU4V/utils/file_handler.py

```diff
@@ -64,7 +64,7 @@ def _convert(s):
     # open the file in universal line ending mode
     with open(file_name, newline='') as infile:
         # read the file as a dictionary for each row ({header : value})
-        reader = csv.DictReader(infile)
+        reader = csv.DictReader(infile, delimiter=',', quotechar='|')
```

Comment:
If we are going to set the delimiter and quote char these should be user configurable in the function input params

Reply:
@odonos12 can you revisit this one.